test/unit/cache/clear.test.js

```diff
@@ -0,0 +1,33 @@
+const mockClient = {
+  del: jest.fn()
```

Comment:
What does del mean here?

Reply:
Delete.  The Redis command -> https://redis.io/commands/del/